tests/Security/TwoFactor/Trusted/JwtTokenEncoderTest.php

```diff
@@ -4,41 +4,47 @@
 
 namespace Scheb\TwoFactorBundle\Tests\Security\TwoFactor\Trusted;
 
-use Lcobucci\JWT\Builder;
+use Lcobucci\JWT\Configuration;
 use Lcobucci\JWT\Signer\Hmac\Sha256;
-use Lcobucci\JWT\Token;
+use Lcobucci\JWT\Signer\Key\InMemory;
+use Lcobucci\JWT\Token\Plain;
 use Scheb\TwoFactorBundle\Security\TwoFactor\Trusted\JwtTokenEncoder;
 use Scheb\TwoFactorBundle\Tests\TestCase;
 
-class JwtTokenEncoderTest extends TestCase
+class AbstractJwtTokenEncoderTest extends TestCase
```

Comment:
I believe this is a leftover from the previous implementation, right? Could be renamed back to `JwtTokenEncoderTest`.